.github/workflows/continuous-integration.yml

```
@@ -1,5 +1,6 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+# Node.js and the exact pnpm version are provisioned by pnpm itself from
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)